cEP-0001.md

```diff
@@ -4,38 +4,101 @@ Decision Making in coala
 |Metadata|                                                                    |
 |--------|--------------------------------------------------------------------|
 |cEP     |0001                                                                |
-|Version |1.1                                                                 |
+|Version |1.2                                                                 |
 |Title   |Decision Making in coala                                            |
-|Authors |Lasse Schuirmann <lasse@gitmate.io>, Mischa Krüger <makman@alice.de>|
+|Authors |Maximilian Scholz <max.scholz@mailbox.org>, Lasse Schuirmann <lasse@gitmate.io>, Mischa Krüger <makman@alice.de>|
 |Status  |Active                                                              |
 |Type    |Process                                                             |
 
+
 Abstract
 --------
 
-This document describes how important decisions affecting the coala community
-may be determined.
+This cEP describes the official roles members of the coala community can hold
+and how they can vote on different levels of decisions.
 
-This cEP describes a voting based decision making level with several levels
-of decisions. Higher levels need more people to agree with the decision.
 
 Roles
 -----
 
-The following roles may be involved in decisions:
+The following roles can be held by members of the coala community
+
+### Newcomers
+
+A newcomer is someone that was just introduced to the coala project and
+community and begins to contribute to it. This rank mainly represents the
+learning phase of the contribution process.
+
+Every community member that started the [newcomer guide](coala.io/newcomer) is
```

Comment:
Hm... maybe just remove the community member. Everyone that got an invitation to the org is a newcomer.